descriptors/ModuleDescriptor-template.json

```diff
@@ -178,7 +178,8 @@
             "circulation.check-in-by-barcode.post"
           ],
           "modulePermissions": [
-            "modperms.circulation.check-in-by-barcode.post"
+            "modperms.circulation.check-in-by-barcode.post",
+            "lost-items-fees-policies.collection.get"
```

Comment:
`modperms.circulation.check-in-by-barcode.post` is a fake permission set which contains all of the module permissions for this endpoint, in order to get around an issue in the infrastructure that I believe is now resolved.

I think we need to be consistent, meaning either we
* add `lost-items-fees-policies.collection.get` to that fake permission set
* inline all of the permissions and remove that fake permission set (eventually I want to do this across the board)

Given this change will likely need to be back ported to 2021 R1, I think we should go with the first option for this change. Apologies, I don't think I realised this was what you had done from the JIRA comments.

Reply:
Okay, I moved  the lost item fee permission to modperms.circulation.check-in-by-barcode.post.